### PR TITLE
New Release 1.1.0

### PR DESCRIPTION
**Pull request by fabiodmota**
Fixed css style guidelines
Fixed whitespaces on helm charts

### CODE REVIEW

**Comment by SebastianBezold** [comment]
Hi @fabiodmota , 
please ask someone to do a content review for all the CSS and React component changes, since i cannot judge, if that is what you are aiming for.
The Chart changes seem fine. So i can approve that part.
Beside that. Just upgrading the version on a bugfix level seems to be quite small, considering you added completely new React components. But again: I cannot tell, if this is only a bugfix or already a new feature.
So please, who ever can do this review for the React parts, point out, if the bugfix version upgrade is ok.

File: src/components/services/EnvironmentService.js [hunk 1]
```diff
@@ -52,6 +52,15 @@ export const getPortalLink = () => {
   return "https://portal.dev.demo.catena-x.net/";
 };
 
+export const getLogoutLink = () => {
+  const hostname = getHostname();
+  if (hostname === "country-risk-dashboard.dev.demo.catena-x.net")
```
Comment (Siegfriedk) [inline src/components/services/EnvironmentService.js:57]: This can't be the right way on how to handle this.

This should be (in my opinion) be configurable from the helm chart itself and not hard coded.

Comment (evegufy) [reply src/components/services/EnvironmentService.js:57]: @fabiodmota 
as discussed in the teams chat, please refer to those as illustrative commits for the approach we used:
https://github.com/catenax-ng/tx-portal-frontend/pull/110/commits/1a990af74a6b071d7d1bafe3247fcda79efc3693
https://github.com/catenax-ng/tx-portal-frontend/pull/110/commits/b1d3301989ec35fa492dcf2ecdb9288146677fbb
Even tough implementation-wise it's not entirely the same, I was inspired with this approach by the traceability team.

Comment (evegufy) [reply src/components/services/EnvironmentService.js:57]: @fabiodmota 
Just to be clear, additionally to the commits above, you also need to enable the env vars in the helm chart.
Reference PR: https://github.com/catenax-ng/tx-portal-cd/pull/5

Comment (fabiodmota) [reply src/components/services/EnvironmentService.js]: Corrected @SebastianBezold , as I said the css would have already been approved and validated by the @evegufy/julia team, now it is also corrected for the hardcoded envs thank for that @evegufy

Comment (fabiodmota) [reply src/components/services/EnvironmentService.js]: Corrected @SebastianBezold , 

Since the new approach i upgraded the version to 1.1.0 

As I said the css would have already been approved and validated by the @evegufy/julia team, now it is also corrected for the hardcoded envs thanks for that @evegufy 

Also added the arq42 documentation approved by Alexander.AK.Keppler@bmw.de

Comment (SebastianBezold) [reply src/components/services/EnvironmentService.js]: Hi @fabiodmota , I cannot find the docs review here. Can you please point me to it, or ask Alexander to document it here in this PR?
@evegufy can you confirm, that the styleguide requirements are now met? cannot find that approval here either.
Please keep these alignments/approvals transparent. For me and anyone else not being part in your private conversations, it is quite hard to follow

Comment (alexKeppler) [reply src/components/services/EnvironmentService.js]: Hi @SebastianBezold, I can confirm the arc42 documentation is up to date.

Comment (evegufy) [reply src/components/services/EnvironmentService.js]: > Hi @fabiodmota , I cannot find the docs review here. Can you please point me to it, or ask Alexander to document it here in this PR? @evegufy can you confirm, that the styleguide requirements are now met? cannot find that approval here either. Please keep these alignments/approvals transparent. For me and anyone else not being part in your private conversations, it is quite hard to follow

@SebastianBezold There is currently a review by Julia going on as part of this ticket: https://jira.catena-x.net/browse/BPDM2-469

Comment (alexKeppler) [reply src/components/services/EnvironmentService.js]: @jjeroch: Could you please give your styleguide approval for QG5 here as requested by @Siegfriedk during the QG5 review.

Comment (SebastianBezold) [reply src/components/services/EnvironmentService.js]: Hi @evegufy @alexsilva-CGI  and @alexKeppler ,
thank you for the input, but please refrain from doing reviews or posting updates in tools, that are not open for the open source community. This makes this whole effort intransparent.

So for this last time, i'll work my way through the comments. For upcoming reviews, please do them transparently for everyone
